src/esp/sim/BatchReplayRenderer.cpp

```diff
@@ -256,6 +257,9 @@ void BatchReplayRenderer::doRender(
                  "with a standalone renderer", );
   static_cast<gfx_batch::RendererStandalone&>(*renderer_).draw();
 
+  // todo: integrate debugLineRender_->flushLines
+  CORRADE_INTERNAL_ASSERT(!debugLineRender_);
```

Comment:
On my "eventually nice to have" TODO list I have a batch-friendly line generator for the new [line shader](https://doc.magnum.graphics/magnum/classMagnum_1_1Shaders_1_1LineGL.html) (right now it's only possible to generate & draw a line mesh as a whole, not gradually add to it). 

Depending on how soon you'd like to have it working for the batch renderer, I can prioritize it.

Reply:
Related FYI, I can't think of a use case where we'll need line-rendering for a multi-tile (multi-env) setup. I.e. thick lines are *only* for showing to humans, not AI model training, because they are fundamentally a UX thing and not photorealistic. They're only for single-tile usage of the batch renderer.